src/main/java/ua/goit/gojava32/kickstarter/factory/DBHelper.java

```
@@ -0,0 +1,124 @@
+
+  public void initDatabase() {
+    final String CONNECTION_STRING = "jdbc:sqlite:kickstarter.db";
+    try {
+      connection = DriverManager.getConnection(CONNECTION_STRING);
+      executeQueryFromFile(DROP_TABLES_FILENAME);
+      executeQueryFromFile(CREATE_TABLES_FILENAME);
+      executeQueryFromFile(INSERT_DATA_FILENAME);
+      //saveImageToDatabase(TEST_IMAGE_FILENAME);
+    } catch (SQLException e) {
+      e.printStackTrace();
+    } finally {
+      if (connection != null) {
+        try {
+          connection.close();
+        } catch (SQLException e) {
+          e.printStackTrace();
+        }
+      }
+    }
+  }
+
+  /*private void saveImageToDatabase(String filename) {
+    File file = new File(filename);
+    Project project = new ProjectDAOImpl().get(1);
+    byte[] bFile = new byte[(int) file.length()];
+    try (FileInputStream fis = new FileInputStream(filename)) {
+      fis.read(bFile);
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+    Image image = FactoryModel.createImage(1, project, bFile);
+    new ImageDAOImpl().add(image);
+  }*/
+
+  private void executeQueryFromFile(String filename) {
+    InputStream input = this.getClass().getResourceAsStream(filename);
+    if (input == null) {
+      throw new RuntimeException("Cannot read file: " + filename);
+    }
+    String query = getStringFromInputStream(input);
+    executeUpdate(query);
+    Logger logger = Logger.getLogger(this.getClass());
+    logger.info("query:"+ query);
+
+    try {
+      input.close();
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+  }
+
+  private static String getStringFromInputStream(InputStream is) {
+    BufferedReader br = null;
+    StringBuilder sb = new StringBuilder();
+
+    String line;
+    try {
+      br = new BufferedReader(new InputStreamReader(is));
+      while ((line = br.readLine()) != null) {
+        sb.append(line);
+      }
+    } catch (IOException e) {
+      e.printStackTrace();
```

no to swallowing exceptions